quiche/src/h3/stream.rs

```diff
@@ -165,11 +165,25 @@ pub struct Stream {
     /// Whether a DATA frame has been received.
     data_received: bool,
 
+    /// Whether a trailing HEADER field is in flight.
+    trailers_in_flight: bool,
+
     /// Whether a trailing HEADER field has been sent.
     trailers_sent: bool,
 
     /// Whether a trailing HEADER field has been received.
     trailers_received: bool,
+
+    /// Tracking for outgoing frame header.
+    pub outgoing_frame_header: Vec<u8>,
```

Comment:
all of these were made `pub` to make adding the feature easier, they can be put behind accessors once we're happy with general approach